src/request.rs

```diff
@@ -277,7 +277,7 @@ pub enum Request {
         /// Incremented version
         version: u64,
         /// Permissions
-        permissions: AppPermissions,
+        token: AuthToken,
     },
```

Comment:
I think we should be renaming the request if we're already getting rid of permissions...

(Which incidentally, maybe should just be named `UpdateAuthAndClientHandlerData` ? As later on this could hold more label info (such as PKs)

Reply:
This was just an `AuthKey` operation(previously). Since we handling tokens now, maybe changing it to `InsAuthToken`, `DelAuthToken` might make more sense I guess? 

For Labels I think we can rename these precisely when we get into the implementation and details that time.

Reply:
Sounds good.

Reply:
Hmm, on second thoughts, I think we can actually leave them unchanged. Because, `AuthKeys` literally mean the App's _Keys_ that are _Authenticated_ by the client. The Token is just piggybacking on this request for further verification use(checking current token & all that) that is actually mapped with the _Authenticated Keys_.

Reply:
Is this request not how we trigger _storing_ of the token though?

Reply:
Storing the _hash_ of the token to be exact.

Reply:
yeh we should probably reflect that then